src/xmipp/libraries/reconstruction/resolution_monogenic_signal.cpp

```diff
@@ -32,558 +32,306 @@ void ProgMonogenicSignalRes::readParams()
-	addParamsLine("  [--vol2 <vol_file=\"\">]: Half volume 2");
-	addParamsLine("  [-o <output=\"MGresolution.vol\">]: Local resolution volume (in Angstroms)");
-	addParamsLine("  [--meanVol <vol_file=\"\">]: Mean volume of half1 and half2 (only it is neccesary the two haves are used)");
-	addParamsLine("  --sym <symmetry>: Symmetry (c1, c2, c3,..d1, d2, d3,...)");
-	addParamsLine("  [--chimera_volume <output=\"Chimera_resolution_volume.vol\">]: Local resolution volume for chimera viewer (in Angstroms)");
-	addParamsLine("  [--sampling_rate <s=1>]   : Sampling rate (A/px)");
-	addParamsLine("                            : Use -1 to disable this option");
-	addParamsLine("  [--volumeRadius <s=100>]   : This parameter determines the radius of a sphere where the volume is");
-	addParamsLine("  [--step <s=0.25>]       : The resolution is computed at a number of frequencies between mininum and");
-	addParamsLine("                            : maximum resolution px/A. This parameter determines that number");
-	addParamsLine("  [--minRes <s=30>]         : Minimum resolution (A)");
-	addParamsLine("  [--maxRes <s=1>]          : Maximum resolution (A)");
-	addParamsLine("  [--trimmed <s=0.5>]       : Trimming percentile");
-	addParamsLine("  [--exact]                 : The search for resolution will be exact (slower) of approximated (fast).");
-	addParamsLine("                            : Usually there are no difference between both in the resolution map.");
-	addParamsLine("  [--noiseonlyinhalves]     : The noise estimation is only performed inside the mask");
-	addParamsLine("  [--filtered_volume <vol_file=\"\">]       : The input volume is locally filtered at local resolutions.");
-	addParamsLine("  [--significance <s=0.95>]    : The level of confidence for the hypothesis test.");
-	addParamsLine("  [--md_outputdata <file=\".\">]  : It is a control file. The provided mask can contain voxels of noise.");
-	addParamsLine("                                  : Moreover, voxels inside the mask cannot be measured due to an unsignificant");
-	addParamsLine("                                  : SNR. Thus, a new mask is created. This metadata file, shows, the number of");
-	addParamsLine("                                  : voxels of the original mask, and the created mask");
-	addParamsLine("  [--automatic]                   : Resolution range is not neccesary provided");
-	addParamsLine("  [--threads <s=4>]               : Number of threads");
+	addUsageLine("MONORES: This algorithm estimate the local resolution map from a single reconstruction");
+	addUsageLine("or two half maps.");
+	addUsageLine("Reference: J.L. Vilas et al, MonoRes: Automatic and Accurate Estimation of ");
+	addUsageLine("Local Resolution for Electron Microscopy Maps, Structure, 26, 337-344, (2018).");
+	addUsageLine("  ");
+	addUsageLine("  ");
+	addParamsLine("  --vol <vol_file=\"\">         : Input map to estimate its local resolution map.");
```

Comment:
`-i` ? "Map whose resolution is to be computed"?

Reply:
--vol is better, because the algorithm can accept two half maps, you can see the optional input of -vol2